package.json

```diff
@@ -50,6 +52,9 @@
     "eslint-plugin-promise": "^2.0.0",
     "eslint-plugin-react": "^6.2.0",
     "eslint-plugin-standard": "^2.0.0",
+    "prop-types": "^15.6.2",
+    "react": "^16.6.3",
+    "react-dom": "^16.6.3",
```

Comment:
hey man dependencies and devDependencies have difference.

@material-ui/core
"prop-types": "^15.6.2",
 "react": "^16.6.3",
  "react-dom": "^16.6.3",
to dependencies (npm install packageName --save)(without flag dev)

Reply:
devDependencies use for eslint, babel, webpack and another developer instruments.